os/booting-on-digitalocean.md

```diff
@@ -7,6 +7,8 @@ schedules per channel. You can [disable this feature][reboot-docs], although we
 don't recommend it. Read the [release notes][release-notes] for specific
 features and bug fixes.
 
+**NOTE: Deploying CoreOS on DigitalOcean is by standard not secure,** because DigitalOcean does not allow private networking. That means that even though you use private networking, you can still be attacked from others inside DigitalOceans private network.
```

Comment:
I wonder if it makes more sense to put this down on 110 near this, since it touches on the "private networking":

> The `$private_ipv4` and `$public_ipv4` substitution variables are fully
supported in cloud-config on DigitalOcean. In order for `$private_ipv4` to be
populated, the droplet must have private networking enabled.

It'd be great to work this into a sentence without having to say "note:". What do you think of this?

> The `$private_ipv4` and `$public_ipv4` substitution variables are fully
supported in cloud-config on DigitalOcean. In order for `$private_ipv4` to be
populated, the droplet must have "private networking" enabled. Private networking isn't available in all regions and isn't actually private from others; instead you can think of it as faster, free bandwidth between your droplets. etcd should be [configured with TLS](https://coreos.com/etcd/docs/latest/security.html#example-3-transport-security-&-client-certificates-in-a-cluster) when using this network.